api/envoy/type/runtime_percent.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package envoy.type;
+
+import "validate/validate.proto";
+import "gogoproto/gogo.proto";
+
+import "envoy/type/percent.proto";
+
+option (gogoproto.equal_all) = true;
+
+// [#protodoc-title: RuntimePercent]
+
+// Runtime derived FractionalPercent with defaults for when the numerator or denominator is not
+// specified via a runtime key.
+message RuntimeFractionalPercent {
```

Comment:
Should this be in `api/envoy/api/v2/core/base.proto` so as to be next to `RuntimeUint32`?

Reply:
I didn't do that because I didn't want to introduce a dependency on percent.proto. It was originally in percent.proto, but an earlier comment from @danielhochman suggested I move it out into its own file.

Reply:
It's fine for base to depend on percent.proto (just not the other way around).